KosherCocoa/Library/Core/Solar/KCNOAACalculator.m

```diff
@@ -91,16 +91,14 @@ - (double)getJulianDay:(NSDate *)date {
     NSDateComponents *components = [self yearMonthAndDayFromDate:date];
     
     NSInteger year = components.year;
-    
     NSInteger month = components.month;
-    
     NSInteger day = components.day;
 
     if (month <= 2) {
         year -= 1;
         month += 12;
     }
-    int a = year / 100;//loses precison but that is how the java port does it
+    int a = (int) year / 100;//loses precison but that is how the java port does it
```

Comment:
Two questions: 
- Is this cast to silence a warning? 
- Can we please add parenthesis around `((int)year)` to improve code clarity?

Reply:
Done